src/Orleans/Messaging/IncomingMessageBuffer.cs

```diff
@@ -128,6 +131,19 @@ public bool TryDecodeMessage(out Message msg)
             headerLength = 0;
             bodyLength = 0;
 
+            return true;
+        }
+
+        void AdjustBuffer()
```

Comment:
this would certainly benefit from some high-level explanation of what it is doing. Also add private modifier to method.